src/components/PostCard/PostCard.js

```diff
@@ -32,8 +32,14 @@ export default class PostCard extends React.Component {
 
   shouldShowDetails = element => {
     if (element === this.refs.postCard) return true
-    if (element.tagName === 'A' || element.tagName === 'LI') return false
+    if (
+      element.tagName === 'A' ||
+      element.tagName === 'LI' ||
+      ['mention', 'topic'].includes(element.getAttribute('data-type'))
+    ) return false
```

Comment:
^^ Catches things before `ClickCatcher` which I like to consolidate such things to. It's kinda awkward but I think it still belongs here.